apiserver/facades/client/bundle/bundle.go

```diff
@@ -67,10 +72,15 @@ func newFacade(ctx facade.Context) (*BundleAPI, error) {
 	authorizer := ctx.Auth()
 	st := ctx.State()
 
+	model, err := st.Model()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
 	return NewBundleAPI(
-		st,
+		NewStateShim(st),
 		authorizer,
-		names.NewModelTag(st.ModelUUID()),
```

Comment:
Why did you change this? Previous implementation was laconic and you did not need to go to .Model().